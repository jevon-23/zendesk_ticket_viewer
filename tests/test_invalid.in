search 102
search hello world
random
help
exit

