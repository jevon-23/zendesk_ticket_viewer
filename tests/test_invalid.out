To get a list of all the tickets, enter "list".
To get an individual ticket, enter "search {subject}" or "search {ticket number}. 
To exit program, enter "exit"

Index is out of range, please give an index that is 0 - 99 

There are no tickets that conatain that subject.

Invalid input. 

Invalid input. 

